### PR TITLE
Provide textual equivalents for a11y

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
#26

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-a11y-alternative-text](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-a11y-alternative-text)

**Comment by Blackbaud-MattGregg** [comment]
@Blackbaud-TrevorBurch Looking good. Could the button and the profile pic text description be flipped in order within file-drop div so the Text description is read out first and then after it is the button to change the picture? Currently, one runs into the button first which I think could be confusing to hear first and wonder what it is referring to instead of hearing the Picture and then the control to change it after.

**Comment by Blackbaud-TrevorBurch** [comment]
@Blackbaud-MattGregg potentially - but that would be a file drop change. I can not modify that as part of this PR as that placement is dictated by the file drop.

If you'd like us to change that would you mind filing an issue here just saying that we should have the file drop's custom contents prior to the trigger button? https://github.com/blackbaud/skyux-forms/issues

This one might get tricky as we don't know what that custom contents are..... so I wonder if there are any cases where we'd want that custom contents to be second. I don't know that there are as that custom content is what is visually rendered "as" the button.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-a11y-alternative-text](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-a11y-alternative-text)

File: src/app/public/modules/avatar/avatar.inner.component.html [hunk 1]
```diff
@@ -6,6 +6,9 @@
     class="sky-avatar-image"
     [hidden]="!src"
   ></div>
+  <div *ngIf="src" class="sky-screen-reader-only">
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/avatar/avatar.inner.component.html]: `class` needs to be on its own line.

File: src/assets/locales/resources_en_US.json [hunk 2]
```diff
@@ -18,5 +18,17 @@
   "skyux_avatar_errormodal_ok": {
     "_description": "Text used on the primary button to dismiss the error modal.",
     "message": "OK"
+  },
+  "skyux_avatar_inner_description": {
+    "_description": "Accessibility info text for screen readers when using an avatar image",
+    "message": "Profile picture of {0}"
+  },
+  "skyux_avatar_upload_change_aria_label": {
+    "_description": "Accesibility text for when an avatar image can be chnaged.",
```
Comment (Blackbaud-AlexKingman) [inline src/assets/locales/resources_en_US.json]: Typo here - `chnaged`.

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Just a couple cleanup items.

File: src/assets/locales/resources_en_US.json (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply src/assets/locales/resources_en_US.json]: Fixed

File: src/app/public/modules/avatar/avatar.inner.component.html (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/avatar/avatar.inner.component.html]: Fixed

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-a11y-alternative-text](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-a11y-alternative-text)

**Review by Blackbaud-MattGregg — APPROVED** [review]

**Pull request merged**